cogs/assistance-cmds/mset9remove.3ds.md

```diff
@@ -0,0 +1,12 @@
+---
+title: mset9remove
```

Comment:
This needs to have proper title. `Manually removing MSET9` or so
help-desc could use some better description. You already did a good job of this one for `mset9crash`, so, follow similar wordings with this command.